app/Console/Commands/VATUSAEventsUpdate.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Event;
+use Carbon\Carbon;
+use GuzzleHttp\Client;
+use Illuminate\Console\Command;
+
+class VATUSAEventsUpdate extends Command {
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'VATUSAEvents:Update';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Fetches events from VATUSA and syncs id_topics in the events table.';
+
+    protected $apiEndpoint = "https://api.vatusa.net/v2/public/events/";
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct() {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle() {
+        $vatusa_events = [];
+        $vatusa_json = $this->getEvents();
+        if (!is_array($vatusa_events)) {
+            return null;
+        }
+        foreach ($vatusa_json as $vatusa_item) {
+            $vatusa_events[$vatusa_item['id_topic']] = $vatusa_item['title'];
+        }
+        $now = Carbon::now();
+        $events = Event::where('status', 1)->get()->filter(function ($e) use ($now) {
+            return strtotime($e->date.' '.$e->start_time) > strtotime($now);
+        });
```

Comment:
I know this isn't the current pattern used most places, but I feel like this logic makes since to live in the Event class.

Reply:
Yeah, I'll add this method to the Event class so we can just call it instead - makes sense so we don't have to keep reusing this code